depends/windows/nasmw/nasmw.txt

```diff
@@ -1 +1 @@
-nasmw http://mirrors.xbmc.org/build-deps/sources/nasm-2.11.06.zip
+nasmw http://www.nasm.us/pub/nasm/releasebuilds/2.12.02/win32/nasm-2.12.02-win32.zip
```

Comment:
shouldn't we host ourself?

Reply:
what for? we are using dependencies already which are hosted on 3rd party servers.

Reply:
okidoki. earlier we wanted to host everything ourself to ensure reproducibility.

Reply:
we always put them on our servers in the past cause other weren't that reliable

Reply:
i just uploaded the file under win32 on the mirrors

Reply:
the URL was updated.

Reply:
why don't we trash this addon. imo this is rather useless because ffmpeg can encode flac too.